src/components/App.js

```diff
@@ -22,13 +24,21 @@ class App extends Component {
     const lat =  Number(user.data.attributes.lat)
     const lng = Number(user.data.attributes.long)
     const truckList = this.createLocationList(user)
-    this.setState({lat: lat, lng: lng, truckList:[...this.state.truckList, ...truckList ]})
+    this.setState({lat: lat, lng: lng, truckList:[...this.state.truckList, ...truckList ], trucks:[...this.state.trucks, ...user.data.attributes.trucks]})
```

Comment:
You can get rid of this method now and update the lat and lng state from the fetch call upon login

Reply:
We aren't receiving lat / long for the user from the backend currently.

Reply:
ok we will need to have the backend team update that return